syft/pkg/conan_metadata.go

```diff
@@ -8,6 +8,9 @@ import (
 type ConanMetadata struct {
 	Name    string `mapstructure:"name" json:"name"`
 	Version string `mapstructure:"version" json:"version"`
+	Options string `mapstructure:"options" json:"options"`
```

Comment:
+ json schema regeneration due to change (might as well wait on this until we know what the final change is)